### ISSUE TITLE
List of all recently added needs from all organizations

### ISSUE THREAD

**Issue by Solisol** [hackforrefugees/bulk-giving#4]

**Comment by Unarmed** [comment]
Is this a giver-facing thing? As in that there should be a publicly visible page containing the X last added needs?

**Comment by Solisol** [comment]
Yes, giver-facing I'm thinking. So think something similar to the current list of organizations, but you also get a list of ten most recent registered needs from organizations.
